packages/framework/lib/handlers/text.ts

```diff
@@ -35,13 +35,15 @@ export default function textHandlerFactory<U extends User<any>>(
 ) {
     function textHandler(
         this: Bot<U>,
-        message: { text: string },
+        message: { text: string; tracking_data?: { [key: string]: string } | string },
         nlp: WitNLP | undefined,
         user: U
     ) {
         const action = matcher.ruleMatcher(message);
         if (action) {
-            return action(user, message.text);
+            // This is needs a check when we update on this version
```

Comment:
If an application already uses ebony with this update it will have to change the way it gets the text parameter. As it now is an attribute `text` in an object.

Reply:
Make sure that the new types will reflect that so that the build will fail without this change when somebody updates the ebony version.